docs/adr/adr-007-monitoring-and-alerting.md

```diff
@@ -0,0 +1,234 @@
+created that contacts engineers out-of-hours should these never succeed in a
+time frame.
+
+For Sidekiq workers that don't affect the user noticeable aspects of Email
+Alert API, such as [EmailDeletionWorker][], we won't raise critical alerts if
+these are failing. Instead we will raise a warning if any Sidekiq workers
+exhaust all their retries and are placed in the [dead set][sidekiq-dead]. This
+approach was chosen as it was a pragmatic technique to monitor multiple workers
+without adding checks for individual workers.
+
+We will modify the [latency checks for email queues][latency-checks] to be
+significantly less sensitive to latency for all email sending except
+[transactional emails][]. This is because transactional emails have an
+[expectation of a prompt delivery][transactional-delivery-expectation] and the
+lack of them blocks a user from progressing with their task. For the other
+queues we will change the time before a warning occurs to be in the measure of
+multiple hours compared to minutes, with the intention that they serve to
+alert a support engineer only when the system is exceptionally busy and may be
+unhealthy. These alerts will only reach a critical status if they hit a 24 hour
+point as this would indicate that we are consistently creating email faster
+than the application can send it, which may mean the system is perpetually
+backlogged.
+
+### Web application
+
+We decided that there should be a mechanism that alerts people out-of-hours if
+the Email Alert API web application appears broken, noting that currently an
+engineer is unlikely to be contacted in the event of the whole Email Alert API
+application becoming unavailable - unless travel advice were published. We
+noted there wasn't a particularly consistent approach to this across GOV.UK
+([with some apps raising an out-of-hours alert when a single instance fails
+healthchecks][publishing-api-alert]) and felt it would be good to take a simple
+approach to avoid surprising engineers.
+
+We decided the most appropriate place to alert would be when [the application
+load balancer has no healthy instances of the application][aws-lb-check]. This
+represents that no instances of the web application are healthy and
+[no requests will succeed][aws-alb-health].
+
+### Alerts to be removed
+
+We will remove the following alerts that will become superseded:
+
+- [Content Changes unprocessed after 2 hours][unprocessed-content-changes]
+- [Messages unprocessed after 2 hours][unprocessed-messages]
+- [Incomplete DigestRuns after 2 hours][incomplete-digest-runs]
+
+## Monitoring Email Alert Service
+
+Email Alert Service listens for [Publishing API][] events and communicates
+these to Email Alert API. We currently [only have alerts][rabbitmq-alert] that
+occur in-hours if this system fails, so there is no visibility for any
+out-of-hours problems.
+
+We intend to improve this by raising an alert that contacts engineers
```

Comment:
```suggestion
We intend to improve and consolidate the existing alerts, and contact an engineer
```

Minor: thought it worth a mention that we're not intending to increase the number of RabbitMQ alerts.